rts/lra/lra-coordinator-jar/pom.xml

```diff
@@ -56,37 +49,211 @@
             <artifactId>arjuna</artifactId>
             <version>${project.version}</version>
             <exclusions>
-                 <exclusion>
-                     <groupId>sun.jdk</groupId>
-                     <artifactId>jconsole</artifactId>
-                 </exclusion>
-             </exclusions>
+                <exclusion>
```

Comment:
formatting?

Reply:
Since this is pre release code this change does not obfuscate the intent of the PR and does not impact anyone wanting to diff the code. After release one it would make sense to minimise the occasion reformatting of blocks.